repository/git_testing.go

```diff
@@ -31,10 +31,11 @@ func CreateTestRepo(bare bool) *GitRepo {
 		log.Fatal(err)
 	}
 
-	if err := repo.StoreConfig("user.name", "testuser"); err != nil {
+	config := repo.LocalConfig()
```

Comment:
That's another reason not to check the version in the constructor. You should be able to feel ok to write `repo.LocalConfig().Store(...)` without performance impact or weird thing happening.